### PR TITLE
feat: dashboard page

### PR DESCRIPTION
**Pull request by geclos**
Dashboard page with projects management

image
image
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,29 @@
...
+> Do not go gentle into that good night.
+>
+> Grave men, near death, who see with blinding sight
+> Blind eyes could blaze like meteors and be gay,
+> Rage, rage against the dying of the light.
+>
+> And you, my father, there on the sad height,
+> Curse, bless, me now with your fierce tears, I pray.
+> Do not go gentle into that good night.
+> Rage, rage against the dying of the light.
```
Comment (andresgutgon) [inline README.md:27]: what's this?

Comment (geclos) [reply README.md:27]: some inspiration

Comment (geclos) [reply README.md:27]: we will remove it when we make the repo public ofc

File: apps/web/src/app/(private)/settings/layout.tsx [hunk 2]
```diff
@@ -1,30 +1,47 @@
...
       navigationLinks={NAV_LINKS}
-      currentUser={session.user}
-      breadcrumbs={[{ name: session.workspace.name }, { name: 'Settings' }]}
+      currentUser={{ ...user }}
+      breadcrumbs={breadcrumbs}
       sectionLinks={sectionLinks}
     >
-      {children}
+      <div className='flex justify-center items-center max-w-[1024px] m-auto pt-8'>
+        {children}
```
Comment (andresgutgon) [inline apps/web/src/app/(private)/settings/layout.tsx:43]: This belongs to ui package

Comment (geclos) [reply apps/web/src/app/(private)/settings/layout.tsx:43]: why? web-ui is only ds components

Comment (geclos) [reply apps/web/src/app/(private)/settings/layout.tsx:43]: I'm merging, let's continue the discussion offline if needed 🙏🏼

Comment (andresgutgon) [reply apps/web/src/app/(private)/settings/layout.tsx:43]: layout elements are ds components. That's my point

**Pull request merged**